e2e/wdio/core/tests/object-status.e2e-spec.ts

```diff
@@ -73,10 +73,11 @@ describe('Object Status test suite', function() {
     });
 
     describe('object status large design examples', function() {
-        xit('should check object status is large', () => {
+        it('should check object status is large', () => {
```

Comment:
think you can just leave it skipped for now since the assertion is skipped (`xit`)